docs/content/importing/_index.md

```diff
@@ -0,0 +1,70 @@
++++
+title="Importing Code"
+weight=2
++++
+
+Code can be imported into joern in two ways. The program `joern-parse`
+can be used to create Code Property Graphs and persist them in a
+format based on Google Protocol Buffer messages, or, `create-cpg` can
+be used to instruct joern-server to create a code property graph and
+make it available via the REST API.
+
+## Creating Code Property Graphs with `joern-parse`
+
+Code property graphs can be created from C/C++ code using
+`joern-parse`. For example, to create a code property graph for the
+vulnerable sample program `tarpit-c`, you can issue the following
```

Comment:
It might be worth clarifying that `tarpit-c` is not a binary program but instead the source root.

Reply:
Done